.changeset/wet-camels-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"@easypost/easy-ui-icons": minor
+---
+
+feat: add AwardStar, HomeWork, LocalPostOffice, Redeem, Timline icons
```

Comment:
**question**: is there a benefit to having changesets on a per project basis?

Reply:
the only benefit is having more specific messages for each project. i was previously lumping all projects into a single message like `create <x /> component` but when looking at the changes for a specific project by itself, they didn't make much sense